src/lune/globals/version.rs

```diff
@@ -1,19 +1,20 @@
 use mlua::prelude::*;
 
 pub fn create(lua: &Lua) -> LuaResult<impl IntoLua<'_>> {
-    let lune_version = format!("Lune {}", env!("CARGO_PKG_VERSION"));
+    let lune_version = format!("{} {}", env!("CARGO_PKG_NAME"), env!("CARGO_PKG_VERSION"));
```

Comment:
Changing the casing here from `Lune` to `lune` is a breaking change, this is why the test case is failing

Reply:
My bad, I didn't actually mean to change that.

Fixed in https://github.com/lune-org/lune/pull/186/commits/ac8d3a944d947ccc4518cf55f7738a8282887c3a.